docs/pages/api-docs/accordion.js

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
-import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+import ApiDocs from 'docs/src/modules/components/ApiDocs';
+import mapApiTranslations, { parsePropsMarkdown } from 'docs/src/modules/utils/mapApiTranslations';
+import jsonPageContent from './accordion.json';
 
-const pageFilename = 'api/accordion';
-const requireRaw = require.context('!raw-loader!./', false, /\/accordion\.md$/);
+export default function Page({ pageContent }) {
+  return <ApiDocs pageContent={pageContent} />;
```

Comment:
Oops, I should have made my feedback clearer:

https://github.com/mui-org/material-ui/blob/f170126a656d5fc66528a5ad3e63c175734b2166/docs/pages/company/jobs.js#L13-L15

https://github.com/mui-org/material-ui/blob/f170126a656d5fc66528a5ad3e63c175734b2166/docs/pages/blog/2020-q1-update.js#L8-L10

Reply:
Inverting it would have required some nasty conditional code